src/components/Header.test.js

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from "@testing-library/react"
+import { BrowserRouter } from "react-router-dom"
+import userEvent from "@testing-library/user-event"
+import Header from "./Header"
+
+describe("<Header />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(
+      <BrowserRouter>
+        <Header />
+      </BrowserRouter>,
+      div
+    )
+  })
+  // it("renders a logo with a src and alt", () => {
+  //   const div = document.createElement("div")
+  //   render(
+  //     <BrowserRouter>
+  //       <Header />
+  //     </BrowserRouter>,
+  //     div
+  //   )
+  //   const logo = screen.getByRole("img")
+  // expect(logo).toHaveAttribute("src", "https://freesvg.org/img/Colorful-Cat-Heart-Tail-Hearts-8.png")
+  // expect(logo).toHaveAttribute("alt", "greetings cat")
+  // })
```

Comment:
Did you need this test?  If so, please comment it back in and make sure it passes.  If not, please remove.